src/EditorFeatures/Core/ValueTracking/ValueTrackingService.cs

```diff
@@ -32,16 +33,32 @@ public ValueTrackingService()
             Document document,
             CancellationToken cancellationToken)
         {
+            using var logger = Logger.LogBlock(FunctionId.ValueTracking_TrackValueSource, cancellationToken, LogLevel.Information);
```

Comment:
I believe if you do not pass this a keyvaluelogmessage nothing will get output to VS telemetry
https://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/Telemetry/VSTelemetryLogger.cs,56

Reply:
That was changed with this PR https://github.com/dotnet/roslyn/pull/52670

Reply:
Ahh sourceroslyn is out of date then, good to know!